infra/growth/src/resources/rules.js

```diff
@@ -793,15 +831,8 @@ class SocialShareRule extends SingleEventRule {
       logger.error(`No identity found for ${ethAddress}`)
       return reward
     }
-    if (
-      !identity.data ||
-      !identity.data.twitterStats ||
-      typeof identity.data.twitterStats.numFollowers !== 'number' ||
-      typeof identity.data.twitterStats.accountAge !== 'number'
-    ) {
-      logger.error(
-        `Missing or invalid twitterStats in identity of user ${ethAddress}`
-      )
+    if (!identity.data || !identity.data.twitterProfile) {
```

Comment:
Is `identity.data.twitterStats` renamed to `identity.data.twitterProfile` here? Or are both different things? 

If it is a rename, I still see [some references of `twitterStats` in this file](https://github.com/OriginProtocol/origin/blob/b8700a463d18179c51c93ccbdaa750d42e8d3a8f/infra/growth/src/resources/rules.js#L859-L867).

Reply:
Yes indeed. Good catch. Fixed !